File: hw/1424/1424.py
def is_prime(num):
    if num <= 1:
        return False
    
    for i in range(2, num // 2 + 1):
        if num % i == 0:
            return False
    
    return True

print("Пары чисел-близнецов:")
for i in range(2, 200):
    if is_prime(i) and is_prime(i+2):
        print(i, "и", i+2)

File: hw/1424/1424.cpp
#include <iostream>
using namespace std;

bool isPrime(int num) {
    if (num <= 1) {
        return false;
    }
    
    for (int i = 2; i <= num / 2; i++) {
        if (num % i == 0) {
            return false;
        }
    }
    
    return true;
}

int main() {
    setlocale(LC_ALL, "");
    cout << "Пары чисел-близнецов: " << endl;
    
    for (int i = 2; i <= 200; i++) {
        if (isPrime(i) and isPrime(i+2)) {
            cout << i << " и " << i+2 << endl;
        }
    }
    
    return 0;
}
